### PR TITLE
Update runtime to 24.08

### PR DESCRIPTION
**Pull request by bztd**
Updated runtime to 24.08
Added necessary fields to .metainfo.xml
Added persistent directory to save changes

In my tests the application runs normally.

### CODE REVIEW

**Comment by bztd** [comment]
@friebetill I have tested the app and it works. Could you merge the changes?

**Comment by bztd** [comment]
Is TubeCards the same project as space?

**Comment by bztd** [comment]
Now the changes made should be saved

**Comment by bztd** [comment]
@hfiguiere Can this pull request be merged?

**Comment by hfiguiere** [comment]
not sure what a "petition" is here.

File: app.getspace.Space.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
   "command": "space",
   "separate-locales": false,
   "finish-args": [
     "--share=network",
     "--socket=wayland",
+    "--share=ipc",
     "--socket=fallback-x11",
-    "--device=dri"
+    "--device=dri",
+    "--persist=."
```
Comment (hfiguiere) [inline app.getspace.Space.json:14]: why persist `.` ?

Comment (bztd) [reply app.getspace.Space.json:14]: The application stores some files and a directory. Downloaded content and user progress are stored.

Comment (hfiguiere) [reply app.getspace.Space.json:14]: you gotta be more specific.

Comment (bztd) [reply app.getspace.Space.json:14]: graphql.hive, graphql.lock and the libCachedImageData directory are stored. Without setting the permission, the work is not preserved.

Comment (hfiguiere) [reply app.getspace.Space.json:14]: what are the *exact path* of each?

Comment (bztd) [reply app.getspace.Space.json:14]: They are all in ${HOME}

Comment (hfiguiere) [reply app.getspace.Space.json:14]: So `graphql.hive` and `graphql.lock` are **files** in the home directory?

Comment (bztd) [reply app.getspace.Space.json:14]: yes

**Pull request merged**